LOAN/LoansSpecific/LoanProducts.rdf

```diff
@@ -55,74 +57,84 @@
 	xmlns:sm="http://www.omg.org/techprocess/ab/SpecificationMetadata/"
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
-	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/LoanProducts/">
+	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansSpecific/LoanProducts/">
 		<rdfs:label xml:lang="en">LoanProducts</rdfs:label>
 		<dct:abstract>This ontology describes loan and mortgage loan products. A product as defined here is something marketed and offered for sale, in this case a loan, such that when a customer takes on the product, they become signatories to a corresponding loan contract. A loan product in this sense is a contractual , off-the-shelf product with terms corresponding to those in the actual contract when it is signed.</dct:abstract>
```

Comment:
This should probably in LoansGeneral if it covers mortgage products

Reply:
@rivettp Let's do that in another pass.